content/en/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Upgrading Windows nodes
+min-kubernetes-server-version: 1.17
+content_template: templates/task
+weight: 40
+---
+
+{{% capture overview %}}
+
+This page explains how to upgrade a Windows node [created with kubeadm](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes).
```

Comment:
add the feature gate please for beta
{{< feature-state for_k8s_version="v1.18" state="beta" >}}